src/Files.App/UserControls/TabBar/TabBar.xaml.cs

```diff
@@ -49,17 +50,11 @@ public bool AllowTabsDrag
 		public Rectangle DragArea
 			=> DragAreaRectangle;
 
-		/// <summary> Starting position when dragging a tab.</summary>
```

Comment:
Please restore the comments

Reply:
Do you really think this is helpful?

Reply:
Yeah, otherwise it's not so clear what it's referring to.